### PR TITLE
feat: add applied_tags parameter to Webhook.send()

### PR DESCRIPTION
**Pull request by tyrantlink**
## Summary

Added applied_tags parameter to Webhook.send(), allowing for forum channel tags to be applied through a webhook.

## Information

- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [x] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [x] I have updated the changelog to include these changes.

### CODE REVIEW

**Review by Dorukyum — CHANGES_REQUESTED** [review]

File: discord/webhook/async_.py [hunk 1]
```diff
@@ -1679,6 +1686,10 @@ async def send(
         thread_name: :class:`str`
             The name of the thread to create. Only works for forum channels.
 
+            .. versionadded:: 2.6
+        applied_tags: List[:class:`Snowflake`]
+            A list of tags to apply to the message. Only works for threads.
+
```
[the remark this answers is not in the document]
Comment (tyrantlink) [reply discord/webhook/async_.py]: Simply applying this suggestion completely broke the formatting, so for now I just changed the versionadded to 2.5

**Comment by Dorukyum** [comment]
Docs should be fine now, will approve after testing the parameter myself just in case.

**Review by Dorukyum — CHANGES_REQUESTED** [review]
I think the parameter should accept a list of `ForumTag`s too to mirror the one in `ForumChannel.create_thread`. Could use a type of `list[Snowflake | ForumTag]` and then take IDs from the tag objects.

**Review by Dorukyum — APPROVED** [review]
I'll implement the suggestion I made earlier in a different pull request, thanks.

**Pull request merged**